### PR TITLE
draft blog post about chatmail 

### PR DESCRIPTION
**Pull request by hpk42**

### CODE REVIEW

File: _posts/2023-12-15-chatmail.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+title: Chatmail - replicable, fast and secure chatting infrastructure for all 
+author: xxx
+image: ../assets/blog/2023-11-green-checkmark-wide.png
+---
+
+![Chatmail logo](../assets/logos/chatmail.png)
+
+Today, we are unvealing **open-signup chatmail services**,
```
Comment (missytake) [inline _posts/2023-12-15-chatmail.md]:
```suggestion
Today, we are unveiling **open-signup chatmail services**,
```

File: _posts/2023-12-15-chatmail.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+- **Convenience**: Get a chatmail address in a few seconds 
+
+- **Privacy**: No questions asked, no name, numbers or e-mail 
+
+- **Speed**: Message delivery in well under a second, end-to-end
+
+Too good to be true? 
+
+You may peek at [nine.testrun.org](https://testrun.org)
```
Comment (missytake) [inline _posts/2023-12-15-chatmail.md]:
```suggestion
You may peek at [nine.testrun.org](https://nine.testrun.org)
```

File: _posts/2023-12-15-chatmail.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+[Nine.testrun.org](https://nine.testrun.org) is operated 
+by some Delta Chat project members, on the side. 
+This mirrors how we envision chatmail services to be operated: 
+by a small group or individuals
+who don't need to worry or spend a lot of time on the service. 
+With some knowledge of DNS, SSH and a spare VPS (get the cheapest you can find),
+you can setup and offer chatmail services pretty quickly. 
+A friend easily recently set it up on a Rasperry Pi for his housing project. 
+Chatmail servers are designed to run with minimal hardware requirements
+and to be easy to tweek even from novice sysadmins or programmers.
```
Comment (missytake) [inline _posts/2023-12-15-chatmail.md]:
```suggestion
and to be easy to tweak even from novice sysadmins or programmers. 
```

File: _posts/2023-12-15-chatmail.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+With some knowledge of DNS, SSH and a spare VPS (get the cheapest you can find),
+you can setup and offer chatmail services pretty quickly. 
+A friend easily recently set it up on a Rasperry Pi for his housing project. 
+Chatmail servers are designed to run with minimal hardware requirements
+and to be easy to tweek even from novice sysadmins or programmers. 
+
+We welcome feedback and contributions to the 
+[Chatmail repository](https://github.com/deltachat/chatmail)
+and intend to do workshops and sessions at both the upcoming
+37c3 Chaos Communication congress end of the year in Hamburg,
```
Comment (missytake) [inline _posts/2023-12-15-chatmail.md]:
```suggestion
[37c3 Chaos Communication congress](https://events.ccc.de) end of the year in Hamburg,
```

File: _posts/2023-12-15-chatmail.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+as well arounnd OFFDEM/FOSSDEM 2024 in Bruxelles.
```
Comment (missytake) [inline _posts/2023-12-15-chatmail.md]:
```suggestion
as well around [OFFDEM](https://offdem.net/)/[FOSDEM](https://fosdem.org/) 2024 in Bruxelles. 
```

File: _posts/2023-12-15-chatmail.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+- **Speed**: Message delivery in well under a second, end-to-end
+
+Too good to be true? 
+
+You may peek at [nine.testrun.org](https://testrun.org)
+or simply scan the below QR code to get a `nine.testrun.org address`: 
+
+<a href="DCACCOUNT:https://nine.testrun.org/cgi-bin/newemail.py">
+    <img width=300 style="float: none;" src="../assets/logos/nine-invite-qrcode.png" /></a>
```
Comment (missytake) [inline _posts/2023-12-15-chatmail.md]:
```suggestion
    <img alt="DCACCOUNT:https://nine.testrun.org/cgi-bin/newemail.py" width=300 style="float: none;" src="../assets/logos/nine-invite-qrcode.png" /></a>
```

**Review by missytake — APPROVED** [review]
I think it's really nice :) I fixed the images and added some typo suggestions.

We need to deploy the echo bot, fix the author and the date before publishing.

File: _posts/2023-12-15-chatmail.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+
+## Send a message to anyone if you can end-to-end encrypt 
+
+Chatmail services don't allow to send un-encrypted messages 
+to anyone outside the service. 
+This makes spammers sad who would try to abuse open-signup chatmail addresses
+because they first would need to setup OpenPGP encryption with all recipients. 
+But Delta Chat uses [Autocrypt](https://autocrypt.org) 
+which does not publish encryption keys anywhere.
+Moreover, today's *Internet platforms do not collect OpenPGP identities*
```
Comment (link2xt) [inline _posts/2023-12-15-chatmail.md]: GitHub does, for example: https://github.com/missytake.gpg
cc @missytake

File: _posts/2023-12-15-chatmail.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+and so can only sell or leak e-mail addresses but not the encryption setup. 
+Therefore it's a massive uphill struggle and economically un-interesting 
+for spammers to abuse chatmail accounts. 
+Also, chatmail services incorporate many best e-mail server practises,
```
Comment (link2xt) [inline _posts/2023-12-15-chatmail.md]:
```suggestion
Also, chatmail services incorporate many best e-mail server practices,
```

File: _posts/2023-12-15-chatmail.md (hunk 5 above)
Comment (link2xt) [reply _posts/2023-12-15-chatmail.md]: French spelling of Bruxelles is on purpose?

Comment (hpk42) [reply _posts/2023-12-15-chatmail.md]: Somewhat, yes.  not a strong pref though.

File: _posts/2023-12-15-chatmail.md [hunk 9]
```diff
@@ -0,0 +1,173 @@
...
+It is not another business model that can replace Whatsapp/Meta,
+at least not without becoming a mirror image. 
+Only communities who sovereignly run their own infrastructures,
+evading centralized control and dependencies, might free themselves. 
+
+The e-mail network is a vast rhizomatic space which surrounds us already,
+and is diversely deployed at planetary scale.
+
+We hope some of you join the amusing journey in exploring it further,
+and put some more long-running critiques to rest :)
```
Comment (missytake) [inline _posts/2023-12-15-chatmail.md]: hmmmm.... not sure about that sentence. It invokes the question "which critiques?" and raises doubts which were not there before.

e.g. one could think "maybe it's moral critique?" - just that we say "put it to rest now", especially without mentioning what critique exactly, doesn't mean that we are right to assume that "we are better now". Quite the opposite, many people/institutions who are still continuing to do wrong are very busy demanding that critique is put to rest.

I'd strike that part completely tbh.

**Review by r10s — APPROVED** [review]

File: _posts/2023-12-15-chatmail.md (hunk 4 above)
Comment (r10s) [reply _posts/2023-12-15-chatmail.md]: i committed  the link directly

File: _posts/2023-12-15-chatmail.md (hunk 5 above)
Comment (r10s) [reply _posts/2023-12-15-chatmail.md]: i committed  the links directly, and i also like the "bruxelles" spelling :)

File: _posts/2023-12-15-chatmail.md (hunk 6 above)
Comment (r10s) [reply _posts/2023-12-15-chatmail.md]: not sure if this alt-text would be helpful; it seems easier to copy+paster the "a href". i'd rather add sth. as "qr code that can be scanned with a mobile device to get an account"

**Comment by r10s** [comment]
ftr: safari/mac did not render the image with the extension `.jpeg` from staging (it works locally) - [using the extension `.jpg`](https://github.com/deltachat/deltachat-pages/pull/780/commits/8955ad7635349b96365e8d075858b755592d09f0), things are fine ... conclusion: always stay conservative with these things :)

**Pull request merged**